Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "cedar-local-agent"
 edition = "2021"
-version = "2.0.0"
+version = "3.0.0"
```

Comment:
Don't bump the version. We have a whole release process for doing this.

Reply:
done